src/utils/transaction.js

```diff
@@ -19,6 +19,7 @@ var convert_1 = require("../utils/convert");
 var constants = require("../constants");
```

Comment:
In this file, it's unnecessary making new 'prepare' function for each new transaction. 
You may refer to https://github.com/virtualeconomy/v-cloud/blob/master/ui/src/builtin_wallet/libs/transaction.js (with only token transfer).
ps.   `a.concat(b.concat(c.concat(.....)))` is too ugly. Why not using imported `concat_1` in this file?

Reply:
The wallet will refactor with [js-v-sdk](https://github.com/virtualeconomy/js-v-sdk) module.  No `prepare` or `a.concat(b.concat(c.concat(.....)))` in SDK. So after refactor, these code will no longer appear.